src/app/Instance/InstancesListPage/InstancesListPage.tsx

```diff
@@ -21,11 +23,11 @@ const InstancesListPage = () => {
       accessor: "name",
       label: t("common.name"),
       formatter: (value: IRowData, row?: IRow) => {
-        const rowId = (row as Instance)?.id;
+        const bridgeId = (row as Instance)?.id;
         return (
           <Link
             data-testid="tableInstances-linkInstance"
-            to={`instance/${rowId}`}
+            to={`/instance/${bridgeId}`}
```

Comment:
I know it pre-existed, but `bridgeId` has the potential to be `undefined` here... although we know it should never happen.

Reply:
Same observation of https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/pull/18#discussion_r847109095